src/EFCore.PG/Metadata/INpgsqlAlterDatabaseOperationAnnotations.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using JetBrains.Annotations;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Metadata
+{
+    /// <summary>
+    /// Represents type-specific annotations specific to Npgsql.
+    /// </summary>
+    public interface INpgsqlAlterDatabaseOperationAnnotations
```

Comment:
You're probably just following the patterns you're seeing, but I don't think there's any use for this interface (or with `INpgsqlDatabaseModelAnnotations`). IIRC the idea of the interface is to provide a read-only interface over the mutable class (although I admit the exact details are hazy at the moment), and since the class is read-only anyway I think we can just remove the interfaces...

Just trying to cut down the amount of code needed for this largely cosmetic refactor.

Reply:
You're spot on—only added to match the landscape... I'll omit these two interfaces before merging.